qa/workunits/rbd/cli_migration.sh

```diff
@@ -99,104 +129,112 @@ compare_images() {
 }
 
 test_import_native_format() {
-    local base_image=$1
-    local dest_image=$2
+    local base_image_spec=$1
+    local dest_image_spec=$2
 
-    rbd migration prepare --import-only "rbd/${base_image}@2" ${dest_image}
-    rbd migration abort ${dest_image}
+    # if image is from namespace
+    local namespace=""
```

Comment:
For consistency with `base_image_spec` and `base_image` variables:
```suggestion
    local base_namespace=""
```

Reply:
updated the variables as per the suggestion.